docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -115,6 +115,23 @@ This document provides a quick overview on how to download and get
 started using WildFly 10 for your application development. For in-depth
 content on administrative features, refer to the WildFly 10 Admin Guide.
 
+[[ee8-preview]]
+=== EE8 Preview ===
+By default, WildFly supports Java EE7. We aim to support Java EE8 in the
+future. As we implement Java EE8 technologies become available we offer a preview of these.
```

Comment:
There's something weird with this sentence.

Reply:
I noticed too :)